erpnext/setup/doctype/company/company.json

```
@@ -736,13 +738,29 @@
    "fieldtype": "Link",
    "label": "Default Use Tax Payable Account",
    "options": "Account"
+  },
+  {
+   "depends_on": "eval:!doc.__islocal",
+   "fieldname": "default_direct_expenses",
+   "fieldtype": "Link",
+   "ignore_user_permissions": 1,
```

```suggestion
   "ignore_user_permissions": 1,
```

Why is there an ignore user permission ?